### PR TITLE
added instructions to get permission to use assets

### PR DESCRIPTION
**Pull request by SteveL-MSFT**

### CODE REVIEW

**Comment by SteveL-MSFT** [comment]
The AppVeyor build [passed](https://ci.appveyor.com/project/PowerShell/powershell/build/6.0.0-beta.7-5594) but somehow it's not linked to this PR

File: assets/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Parties interested in using these logos can review the [Use of Microsoft Copyrighted Material](https://www.microsoft.com/en-us/legal/intellectualproperty/permissions) page.
+If the use falls into any of those categories, you can move forward with the use and will not need additional permission.
+
+## Third Party Permission Requests
+
+If questions remain after reviewing the permitted uses page, please submit a request to [Third Party Permissions](mailto:mscrqs@microsoft.com).
+The people supporting this alias will direct the request depending upon the content.
+
+If a request needs to be submitted to the [Third Party Permissions](mailto:mscrqs@microsoft.com) alias, the request should include the following:
```
Comment (TravisEz13) [inline assets/README.md]: This isn't an alias in this usage.

File: assets/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Permitted Uses
+
+Parties interested in using these logos can review the [Use of Microsoft Copyrighted Material](https://www.microsoft.com/en-us/legal/intellectualproperty/permissions) page.
+If the use falls into any of those categories, you can move forward with the use and will not need additional permission.
+
+## Third Party Permission Requests
+
+If questions remain after reviewing the permitted uses page, please submit a request to [Third Party Permissions](mailto:mscrqs@microsoft.com).
+The people supporting this alias will direct the request depending upon the content.
```
Comment (TravisEz13) [inline assets/README.md]: This isn't an alias in this usage.

**Review by TravisEz13 — APPROVED** [review]
Two minor comments

File: assets/README.md (hunk 1 above)
Comment (SteveL-MSFT) [reply assets/README.md]: will remove the word `alias`

File: assets/README.md (hunk 2 above)
Comment (SteveL-MSFT) [reply assets/README.md]: will remove the word `alias`

**Pull request merged**